guide/build-config.md

```diff
@@ -554,6 +554,9 @@ The `GENERATE_REDHAT_DEPENDENCIES_EXTENDED` strategy gets same result as `GENERA
 
 All `GENERATE*` strategies require that the repository generation is **not** ignored.
 
+`Optional parameters available in all strategies:`
+- `excludeSourceBuilds` - used to specify the list of build configurations that you don't want to be included in the generated sources.
+
```

Comment:
can you please add example or specify in which format users could specify this artifacts? f.e.: 
`
'externalAdditionalArtifacts' - artifacts from other builds, not within this product's build group. This list should contain artifact identifiers, as shown in the PNC UI. `

Reply:
@jomrazek Thanks for reviewing it. 

I added an example for the `excludeSourceBuilds` option for  `sourcesGeneration` and also for `repositoryGeneration` (as it did not exist yet).